src/views/result.js

```diff
@@ -0,0 +1,70 @@
+
+import { quizData } from '../data.js';
+import {
+  USER_INTERFACE_ID,
+} from '../constants.js';
+
+export const  result= () => {
+// RESET THE DOM--------------------------------------
+  const userInterface = document.getElementById(USER_INTERFACE_ID);
+  userInterface.innerHTML = '';
+// ====================================
+
+// CREATION OF RESULT DOM ELEMENTS ----------------
+
+const result = document.createElement('div');
+const percentage = document.createElement('div');
+const resultNum = document.createElement('div');
+const numContainer = document.createElement('div');
+const numContainer2 = document.createElement('div');
+const score = document.createElement('img');
+const complement = document.createElement('h1');
+const complement2 = document.createElement('h1');
```

Comment:
The naming can be improved here. `Complement2` does not help me out much.

Reply:
impoved the name to something descriptive...
just fixed...